ts/application.ts

```diff
@@ -160,21 +159,9 @@ export class Application {
     async ensureBucket(args: { bucketName: string }) {
         const client = await this.getBucketsClient()
 
-        // Version 1
-        // const roots = await client.list();
-        // const existing = roots.find((bucket) => bucket.name === args.bucketName)
-        // if (existing) {
-        //     return existing.key;
-        // }
-
-        // const created = await client.init(args.bucketName);
-        // const bucketKey = created.root ? created.root.key : ''
-
-        // Version 2
-        const result = await client.open(args.bucketName)
-        const bucketKey = result?.key
+        const result = await client.getOrInit(args.bucketName)
```

Comment:
the new `getOrInit` function is designed for your ensure method. Unfortunately this name is changing in the next release to getOrCreate, I'll try to PR again when that happens.